angular-ngrx-scss/src/app/repository/components/repo-header/repo-info/repo-info.component.scss

```diff
@@ -0,0 +1,28 @@
+@use '../../../../shared/styles/variables.scss';
+
+.info-container {
+  display: flex;
+  gap: 0.9rem;
+  .watch,
```

Comment:
Maybe we can use one class for these buttons as they share the same styles and use the BEM conventions here?

Something like: 

``` SCSS
.info {
  ....
  &__counter {
  ...
    &__icon {
    ...
    }
    &__count {
    ...
    }
  }
}
```